src/main/kotlin/org/b12x/gfe/core/model/parsers/ParserVersionData.kt

```diff
@@ -0,0 +1,51 @@
+package org.b12x.gfe.core.model.parsers
+
+import com.beust.klaxon.JsonArray
+import com.beust.klaxon.Parser
+import okhttp3.Response
+import org.b12x.gfe.utilities.FileManagement
+import org.b12x.gfe.utilities.DirectoryManagement
+import java.io.File
+
+object ParserVersionData {
+
+    private val directoryManagement = DirectoryManagement()
+
+    // HLA Version API
+    const val DB_VERSIONS = "http://gfe.b12x.org/v1/imgt-versions"
```

Comment:
Consider declaring all of the related string constants in a single file. https://stackoverflow.com/a/48820895

I don't know if that's the right thing to do here, since I don't understand the purpose of this constant.

Reply:
Maybe I should move it to the top of the file (another suggestion in the link you gave.) The URLs are the access points for the data, and would only be used in this one file. I've always struggled with where to put them, but the only place I've ever consistently looked to find them is in the file they're used in.